specificatie/openapi.yaml

```diff
@@ -283,7 +283,7 @@ paths:
         - Ingeschreven Personen
   /ingeschrevenpersonen/{burgerservicenummer}/kinderen/{id}:
     get:
-      operationId: ingeschrevenpersonenBurgerservicenummerkinderenId
+      operationId: getIngeschrevenPersonenBurgerservicenummerKinderenId
```

Comment:
Kan getIngeschrevenPersonenBurgerservicenummerKinderenId => GetKind? Dit is wat je met deze path wil bereiken